bin/all_sky_search/pycbc_fit_sngls_over_multiparam

```diff
@@ -404,10 +404,12 @@ for param, slog in zip(args.fit_param, args.log_param):
     else:
         raise ValueError("invalid log param argument, use 'true', or 'false'")
 
-nabove_smoothed = []
-alpha_smoothed = []
-ntotal_smoothed = []
-rang = numpy.arange(0, len(nabove))
+n_templates = len(nabove)
```

Comment:
is there a reason why the previously defined `num_templates` should not be used?